spec/requests/user_badges_controller_spec.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
-require 'rails_helper'
+require "rails_helper"
```

Comment:
can you remove all the formatting changes here please? It's not the purpose of the PR and makes it less readable.